service/gcs/core/gcs/cleanup.go

```diff
@@ -49,35 +50,35 @@ func (c *gcsCore) CleanupContainer(id string) error {
 }
 
 // forceDeleteContainer deletes the container, no matter its initial state.
-func (c *gcsCore) forceDeleteContainer(id string) error {
-	exists, err := c.Rtime.ContainerExists(id)
+func (c *gcsCore) forceDeleteContainer(container runtime.Container) error {
```

Comment:
`runtime.Container` is an interface backed by `*runc.Container`, so it is by ref.